src/entries/popup/pages/settings/customChain/index.tsx

```diff
@@ -457,6 +457,14 @@ export function SettingsCustomChain() {
           name: symbol,
         },
         rpcUrls: { default: { http: [rpcUrl] }, public: { http: [rpcUrl] } },
+        blockExplorers: {
+          default: {
+            name: customRPC.explorerUrl
+              ? getDappHostname(customRPC.explorerUrl)
+              : '',
+            url: customRPC.explorerUrl || '',
```

Comment:
if there is no explorerUrl should we do `''`? can that show something like `view on  ` like "view on >blank<" anywhere?

Reply:
We already have checks on the UI side for this. I think wagmi types were complaining about undefined